system/security/LdapSecurity/ldapsecurity.cpp

```diff
@@ -548,6 +555,8 @@ CLdapSecManager::CLdapSecManager(const char *serviceName, IPropertyTree &config)
 
 CLdapSecManager::~CLdapSecManager()
 {
+    if (!m_cfg->getPropInt("@sharedCache", true))
```

Comment:
Changed to getPropBool in 538 and removed the unneeded variable, and changed it here as well